updateHostsFile.py

```diff
@@ -21,15 +21,16 @@
 import time
 from glob import glob
 
-import lxml  # noqa: F401
-from bs4 import BeautifulSoup
+try:
+    import requests
+except ModuleNotFoundError as e:
```

Comment:
```suggestion
except ModuleNotFoundError:
```

I think this should fix the linting errors.

Reply:
Oops I hadn't refreshed the page, it should be fixed now.